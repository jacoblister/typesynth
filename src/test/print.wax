(func printTest
  (print "printTest:")

  (print "abc")
  (print 123)
  (print 123.4)
  (print (+ 2 3))
  (print (+ 10.0 1.1))

  (let value int 999)
  (print value)

  (let msg str "xyz")
  (print msg)

  ; (print (get msg 1))
  (let out str (alloc str ""))
  (<< out (get msg 1))
  (print out)
)

(func main (result int)
  (call printTest)
  (return 0)
)