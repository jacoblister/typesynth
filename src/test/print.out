printTest:
abc
123
123.4
5
11.1
999
xyz
y
